pkg/client/preflight.go

```diff
@@ -54,20 +54,31 @@ const (
 	kubeSystemNamespace = "kube-system"
 	kubeDNSLabelKey     = "k8s-app"
 	kubeDNSLabelValue   = "kube-dns"
+	coreDNSLabelValue   = "coredns"
 )
 
 func preflightDNSCheck(client kubernetes.Interface, cfg *PreflightConfig) error {
-	selector := metav1.AddLabelToSelector(&metav1.LabelSelector{}, kubeDNSLabelKey, kubeDNSLabelValue)
+	var dnsLabels = []string{
+		kubeDNSLabelValue,
+		coreDNSLabelValue,
+	}
 
-	obj, err := client.CoreV1().Pods(kubeSystemNamespace).List(
-		metav1.ListOptions{LabelSelector: metav1.FormatLabelSelector(selector)},
-	)
-	if err != nil {
-		return errors.Wrap(err, "could not retrieve list of pods")
+	var nPods = 0
+	for _, labelValue := range dnsLabels {
```

Comment:
Can you construct a set based label selector https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement  ?  I'd have to dig the upstream test cases for `valueA or valueB in key`

Reply:
So ran through and I guess you can't.